compiler/luci/export/src/ExportMetadata.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_EXPORT_METADATA_H__
+#define __LUCI_EXPORT_METADATA_H__
+
+#include <cassert>
+#include <map>
+#include <string>
+#include <set>
+#include <vector>
+
+namespace luci
+{
+
+class ExportMetadata
+{
+public:
+  ExportMetadata() = default;
+
+public:
+  void link_source(uint32_t node_id, uint32_t source_id);
+  void add_source(uint32_t source_id, std::string origin_name);
```

Comment:
Can you add a comment or readme file how node_id, source_id and origin_name is converted to MetaData?